src/Helpers/Components.php

```diff
@@ -420,6 +451,14 @@ function ($item) use ($attributes, $key) {
 				))[0]['variable'] ?? null;
 
 				$value = $selectVariable === null ? $attributes[$key] : $selectVariable;
+
+				if ($value === "") {
+					continue;
+				}
+
+				if (gettype($value) === 'array') {
```

Comment:
```suggestion
				if (is_array($value)) {
```

Cleaner, and marginally faster 😄